src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -213,7 +213,7 @@ private static void updateTrackingInfo(String statusType, String oldType, String
                     logger.info("Placed CE order for kit with external order number " + kit.getExternalOrderNumber());
                     kit.changeCEOrdered(true);
                 }
-                else if (shouldTriggerKitDeliveryEvent(statusType, oldType)) {
```

Comment:
Previous update used the same logic for kits going to participant and kits coming back.  We need to different criteria: one for the outbound kit being in transit (or delivered) and another one for the kit being delivered back at broad (but not just in transit).